pkg/client/monitoring/v1/types.go

```diff
@@ -49,6 +49,10 @@ type PrometheusList struct {
 // Specification of the desired behavior of the Prometheus cluster. More info:
 // http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#spec-and-status
 type PrometheusSpec struct {
+	// Standard object’s metadata. More info:
+	// http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	// Metadata Labels and Annotations gets propagated to the prometheus pods.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Just to avoid confusion, this should probably be called `PodMetadata` and the json tag `podMetadata`.